tests/gocase/unit/type/tdigest/tdigest_test.go

```diff
@@ -310,4 +311,66 @@ func tdigestTests(t *testing.T, configs util.KvrocksServerConfigs) {
 		infoAfterEmptyReset := toTdigestInfo(t, rsp.Val())
 		require.EqualValues(t, 100, infoAfterEmptyReset.Compression)
 	})
+	t.Run("tdigest.quantile with different arguments", func(t *testing.T) {
```

Comment:
Hi @SharonIV0x86 ,

We could also add an unordered sequence and an empty tdigest as cases. 😊

Best Regards,
Edward